examples/webgl_raycast_sprite.html

```diff
@@ -0,0 +1,177 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>three.js webgl - raycast - sprite</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+        <style>
+            body {
+                margin: 0px;
+                background-color: #fff;
+                overflow: hidden;
+            }
+
+            #info{
+                position: absolute;
+                z-index: 1;
+                width: 100%;
+                text-align: center;
+            }
+        </style>
+    </head>
+<body>
+    <div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl raycast sprite<br></div>
+    <script src="../build/three.js"></script>
+    <script src="./js/controls/OrbitControls.js"></script>
+    <script>
+        var renderer;
+        var scene;
+        var spriteGroup;
+        var camera;
+        var controls;
+
+        init();
+
+        function init() {
+
+            initRender();
+            initScene();
+            initCamera();
+            addSprite();
+
+            render();
+            window.addEventListener( "mousemove", onDocumentMouseMove, false );
+
+        }
+
+        function render() {
+
+            renderer.render( scene, camera );
+            requestAnimationFrame( render );
+
+        }
+
+        function initRender() {
+
+            renderer = new THREE.WebGLRenderer( {
+                antialias: true
+            } );
+            renderer.setSize( window.innerWidth, window.innerHeight );
+            document.body.appendChild( renderer.domElement );
+            renderer.setClearColor( 0xFFFFFF, 1.0 );
+
+        }
+
+        function initScene() {
+
+            scene = new THREE.Scene();
+            spriteGroup = new THREE.Object3D();
+            scene.add( spriteGroup );
+
+        }
+
+        function initCamera() {
+
+            camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 1, 1000 );
+            camera.position.y = 15;
+            camera.position.z = 15;
+            camera.position.x = 15;
+            camera.lookAt( {
+                x: 0,
+                y: 0,
+                z: 0
+            } );
```

Comment:
This is no good approach since you "fake" a `Vector3` object. Do this instead: `camera.lookAt( scene.position )`;